src/pages/Stream/index.tsx

```diff
@@ -58,6 +63,19 @@ const Stream: FC = () => {
 		isRefetching: isSchemaRefetching,
 	} = useGetStreamSchema({ streamName: currentStream || '' });
 
+	useEffect(() => {
+		const updatedTimeRange = getDefaultTimeRange();
+		setLogsStore((store) => {
```

Comment:
Let's create a fn in LogsProvider. This action is needed everywhere we use time range.

Reply:
Addressed this review comment.